### ISSUE TITLE
Null activities and unbounded activities are indistinguishable

### ISSUE THREAD

**Issue by pmdoll** [dollabs/planviz#68]
![activity-bounds](https://user-images.githubusercontent.com/14317443/27392084-ab5c3ee8-5673-11e7-8463-6c07a64d668c.png)
Null activities are assumed to have [0 0] constraints while unbounded activities have [0 :infinity]. Since they have different semantics, they should be visually distinguishable.

**Comment by dcerys** [comment]
I agree.  The ability to distinguish null activities from non-null activities goes beyond the default temporal constraint.  It's the fundamental semantic difference between a null activity (which isn't an activity) and a real activity that is even more important.  For the observer of the TPN, they both appear as similar (often identical) arcs between nodes.  

Often, null activities are drawn slightly differently, in that non-null activities are (always?) drawn horizontally while null activities are drawn at arbitrary angles.  However, that's pretty subtle.  Some other suggestions to distinguish null activities from non-null:

- use a dashed line for the arc
- use a thinner line for the arc
- use an explicit "NULL" (or some other) activity label for all null activities [not a great idea, since it will clutter the TPN]

Let's NOT distinguish the null activities by color, since that would interfere with the normal expected coloring of execution state.

**Comment by pmdoll** [comment]
Null activities are effectively connectors of 2 nodes and they exist for historical reasons. Cytoscape based planviz used edges without arrows to indicate null activities and effectively distinguished null from non-null activities.
